stdlib/public/Concurrency/Task.swift

```diff
@@ -167,8 +167,14 @@ extension Task {
   ///
   /// - Returns: The task's result.
   public var value: Success {
-    get async throws {
-      return try await _taskFutureGetThrowing(_task)
+    // FIXME: This seems wrong that abi test is not freaking out here?
+    //        At the same time, adding a silgen_name compat accessor with old signature results in duplicate definitions -- is mangling of typed throws not done properly on return position?
+    get async throws(Failure) {
```

Comment:
This seems weird, see the FIXME here ^

It seems like the typed throw isn't mangled differently than a throw? I was under the impression it should. 
Adding an overload and the usual `__abi_` dance with a silgen_name of the previous name ends up with duplicate declarations of $sScT5valuexvg (see in first [commit](https://github.com/apple/swift/pull/74110/commits/ebc3d23c01db140c51a588e3d8bc413d39cc6ed6))